source/kraken/realtime.cpp

```diff
@@ -120,10 +136,15 @@ static std::ostream& operator<<(std::ostream& s, const nt::StopTime& st) {
  */
 static bool check_disruption(const nt::disruption::Disruption& disruption) {
     auto log = log4cplus::Logger::getInstance("realtime");
+    using nt::disruption::StopTimeUpdate;
     for (const auto& impact: disruption.get_impacts()) {
         boost::optional<const nt::StopTime&> last_st;
         for (const auto& stu: impact->aux_info.stop_times) {
             const auto& st = stu.stop_time;
+            bool deleted = (in(stu.arrival_status,
+            {StopTimeUpdate::Status::DELETED, StopTimeUpdate::Status::DELETED_FOR_DETOUR}) ||
```

Comment:
Same here, I think it's better to separate stop_time events as we check consistency separately.